get-started/versioning-availability.md

```diff
@@ -4,168 +4,209 @@ mapped_pages:
-
-Features may have different availability states between:
-
-- **Deployment type**: The environment where the feature is available (Stack, Serverless, ECE, ECK, etc.)
-- **Lifecycle state**: The development or support status of the feature (GA, Beta, etc.)
-- **Version**: The specific version the lifecycle state applies to
-
-#### Lifecycle states
-
-| State | Description |
-|-------|-------------|
-| **Generally Available (GA)** | Production-ready feature (default if not specified) |
-| **Beta** | Feature is nearing general availability but not yet production-ready |
-| **Technical preview** | Feature is in early development stage |
-| **Unavailable** | Feature is not supported in this deployment type or version |
-
-### Where feature availability may differ
-
-Features may have different states between:
-
-- **[Elastic Stack](the-stack.md)** versions (e.g., 9.0, 9.1)
-- **Deployment types** (and deployment versions)
-  - [Elastic Cloud Hosted](/deploy-manage/deploy/elastic-cloud/cloud-hosted.md)
-  - [Elastic Cloud Serverless](/deploy-manage/deploy/elastic-cloud/serverless.md)
-  - [Self-managed deployments](/deploy-manage/deploy/self-managed.md)
-  - [Elastic Cloud Enterprise (ECE)](/deploy-manage/deploy/cloud-enterprise.md)
-    - ECE deployment versions (for example, 4.0.0)
-  - [Elastic Cloud on Kubernetes (ECK)](/deploy-manage/deploy/cloud-on-k8s.md)
-    - ECK deployment versions (for example, 3.0.0)
-- **Serverless project types**
-  - Security
-  - Elasticsearch
-  - Observability
-
-### Important tips when reading the docs
+Starting with {{stack}} 9.0.0, Elastic no longer publishes separate documentation sets for each minor release. Instead, all changes in the 9.x series are included in a single, continuously updated documentation set.
 
-- Always check feature lifecycle state for your specific deployment type and version
-- Pay attention to Elastic Stack version requirements
-- Note that Serverless features may vary by project type
+This approach helps:
+* Reduce duplicate pages
+* Show the full history and context of a feature
+* Simplify search and navigation
 
-### Availability badges in the docs
+### How do I know content was added in a specific version?
 
-Our documentation uses badges to help you quickly identify where and when features are available for your specific environment.
+We clearly mark content added or changed in a specific version using availability badges. The availability badges appear in page and section headers. 
 
-Badges can appear in two places:
-1. **Page headers**: Shows the overall availability across all deployment types
-2. **Section headers**: Indicates specific availability for content in that section
-
-### How to read the badges
-
-Here are some examples to help you understand how to read the availability badges.
-
-#### Example #1: Stack-only feature
+#### Elastic Stack example
 
 ```yaml {applies_to}
 stack: ga 9.1
 ```
```

Comment:
Should these examples instead be rendered `applies_to` badges? That would better help users map what they see visually in the docs to what it means.

for example:

````suggestion
```{applies_to}
stack: ga 9.1
```
````